setup.py

```diff
@@ -19,14 +19,15 @@
 import versioneer
 
 
-REQUIRES = ["numpy>=1.16", "h5py", "PyYAML", "cython", "future", "click"]
```

Comment:
Looks like you need to add `click` to the requirements file now

Reply:
Yeah. Sorry. That's done now.